src/main/java/datawave/query/util/AllFieldMetadataHelper.java

```diff
@@ -90,6 +91,8 @@ public class AllFieldMetadataHelper {
     protected final TypeMetadataHelper typeMetadataHelper;
     protected final CompositeMetadataHelper compositeMetadataHelper;
     
+    protected final TypeFactory typeFactory = new TypeFactory();
```

Comment:
Can we potentially add the ability to configure the size and ttl here?

Reply:
Sure...let me work up a test or two for that..